app/routes/application.js

```diff
@@ -1,12 +1,28 @@
 import Ember from 'ember';
 import ApplicationRouteMixin from 'simple-auth/mixins/application-route-mixin';
 
+const { inject } = Ember;
+const { service } = inject;
+
 export default Ember.Route.extend(ApplicationRouteMixin, {
-  flashMessages: Ember.inject.service(),
+  flashMessages: service(),
+
   actions: {
     sessionInvalidationSucceeded(){
       this.get('flashMessages').success('auth.confirmLogout');
       this.transitionTo('login');
+    },
+
+    error(error, transition) {
+      transition.abort();
+
+      this.get('flashMessages').alert('general.transitionErrorMessage');
+
+      // Manage route-related errors
```

Comment:
Ok - it needs some comments then for future developers otherwise it is just commented out code.  Or you can remove it and whoever comes after will have to figure it out, which is OK too since they will need to learn this stuff to change it.